CppCoreGuidelines.md

```diff
@@ -7466,7 +7466,7 @@ Problems:
 and all classes derived from `Shape` and all code using `Shape` will need to be reviewed, possibly changed, and probably recompiled.
 
 The implementation of `Shape::move()` is an example of implementation inheritance:
-we have defined `move()` once and for all for all derived classes.
+we have defined `move()` once and for all derived classes.
```

Comment:
not a typo, though perhaps a comma would've made it more obvious

Reply:
No, It's Not a Typo